assets/css/_draft_picks.scss

```diff
@@ -46,3 +46,29 @@
     }
   }
 }
+
+.draft-summary-table {
+  margin-bottom: 2em;
+  
+  td,
+  th {
+    padding: 1px;
+    font-size: $small-font-size;
+
+    @include media ($large-screen) {
+      font-size: $base-font-size;
+    }
+  }
+  th {
+    text-align: center;
+  }
+  td {
+    text-align: center;
+  }
+  th:nth-child(1) {
+    text-align: left;
+  }
```

Comment:
Rule declaration should be followed by an empty line

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#emptylinebetweenblocks)